xbmc/cores/VideoPlayer/DVDSubtitles/DVDSubtitleParser.h

```diff
@@ -34,7 +33,9 @@ class CDVDSubtitleParserCollection
 {
 public:
   explicit CDVDSubtitleParserCollection(const std::string& strFile) : m_filename(strFile) {}
-  ~CDVDSubtitleParserCollection() override = default;
+
+  ~CDVDSubtitleParserCollection() { m_collection.Clear(); }
+
```

Comment:
You can keep the defaulted destructor, `m_collection` already calls `Clear()` in its destructor.

Reply:
Oh, I didn't go that deep :) thanks for noticing. Will change.
Will also keep the override for the defaulted destructor.